src/aws/osml/tile_server/services/models.py

```diff
@@ -1,15 +1,13 @@
 #  Copyright 2023-2024 Amazon.com, Inc. or its affiliates.
 
-import logging
 from enum import auto
 from typing import List, Optional
 
 from pydantic import BaseModel, Field
 
 from aws.osml.gdal import RangeAdjustmentType
-from aws.osml.tile_server.utils import AutoLowerStringEnum, AutoUnderscoreStringEnum
 
-logger = logging.getLogger("uvicorn")
+from ..utils.string_enums import AutoLowerStringEnum, AutoUnderscoreStringEnum
```

Comment:
not 100% sure but I think the better way to do this is with the full path to the import?

Reply:
I'll update the PR to make it consistent with the other imports